### ISSUE TITLE
Add Article for File IO in C++

### ISSUE THREAD

**Issue by jrg94** [TheRenegadeCoder/sample-programs#406]
See #405 
Thanks, @Noah11012!

**Comment by jrg94** [comment]
@Noah11012 if you accept the invite to the org, I'll get this assigned to you.

**Comment by Noah11012** [comment]
I'm working on the article now.

**Comment by jrg94** [comment]
You have to accept The Renegade Coder org invite that got emailed to you.

**Comment by Noah11012** [comment]
I think I'm already invited.

**Comment by jrg94** [comment]
You should be, yes. But, I think you have to accept that invite (from GitHub). Otherwise, your name doesn't appear in the assign menu when I try to assign this to you.

**Comment by Noah11012** [comment]
Accepted.

**Comment by jrg94** [comment]
Perfect!

**Comment by jrg94** [comment]
Took a quick look at your article, and it looks great! Nice job 👍

**Comment by jrg94** [comment]
For reference, we're looking to publish your article on October 29th, 2018. I'll have to write up the opening article for the 26th.

**Comment by Noah11012** [comment]
Opening article for the new File I/O series?

**Comment by jrg94** [comment]
Yeah, each series has an opening article which explains the series and has links to all the containing articles (i.e. https://therenegadecoder.com/code/hello-world-in-every-language/)

**Comment by Noah11012** [comment]
Oh, okay.

**Comment by jrg94** [comment]
Hey, Noah! I made a few edits to your article. Take a look to see how you like them. Feel free to make changes as well.
